docs/libmamba-vs-classic.md

```diff
@@ -14,18 +21,27 @@ If the given solution is not fully satisfying, try to restrict your request a bi
 For example, if you run `conda install scipy` and do not get the latest version, try using a more explicit command: `conda install scipy=X.Y`.
 ```
 
-You should probably still use `classic` in the following scenarios:
-
-* Backwards compatibility is very important in your use case, and you need your environments solved in the same way they have been solved in the past years.
-  This is a bit more important in long-lived environments than new ones.
+You might still need `classic` here and there, specially if backwards compatibility is very important in your use case, and you need your environments solved in the same way they have been solved in the past years.
+This is a bit more important in long-lived environments than new ones.
 
 ```{important}
 You can always use `--solver=classic` to re-enable the `classic` solver temporarily for specific operations, even after setting `libmamba` as default.
 ```
 
-## Differences between `libmamba` and `classic`
+## Intentional deviations from `classic`
+
+With the release of `conda-libmamba-solver`, we took the opportunity to improve some aspects of the solver experience that were not possible to change in `classic` due to backwards compatibility restraints. The main ones are:
+
+* `conda-libmamba-solver` does not use `current_repodata.json` by default. Instead, it always uses the full `repodata.json` files.
+* `conda-libmamba-solver` does not retry with `--freeze-installed` by default. Instead, it has a tighter retry logic that progressively relaxes the constraints on the conflicting packages.
+* `conda-libmamba-solver` does not allow the user to override the configured pinned specs by specifying other constraints in the CLI. Instead, it will error early. To override pinned specs,
```

Comment:
```suggestion
* `conda-libmamba-solver` does not allow the user to override the configured [pinned specs](https://docs.conda.io/projects/conda/en/stable/user-guide/tasks/manage-pkgs.html?highlight=pinned#preventing-packages-from-updating-pinning) by specifying other constraints in the CLI. Instead, it will error early. To override pinned specs,
```

It might be nice to link to the documentation on pinned specs because it's not a very well known feature.